### ISSUE TITLE
metadata.json permissions are too strict in v2.1.11

### ISSUE THREAD

**Issue by TimAtLassian** [example42/puppet-iptables#72]
The `metadata.json` file in latest release of `puppet-iptables` is only readable by the owner:

```
-rw-r--r--@ 1 tcinel  staff   281  5 Apr  2014 Gemfile
-rw-r--r--@ 1 tcinel  staff   674 25 Sep  2013 LICENSE
-rw-r--r--@ 1 tcinel  staff  7296 13 Feb  2015 README.md
-rw-r--r--@ 1 tcinel  staff   785 11 Sep  2014 Rakefile
-rw-------@ 1 tcinel  staff  2329 18 Aug 23:27 checksums.json
drwxr-xr-x@ 3 tcinel  staff   102 18 Aug 23:27 lib
drwxr-xr-x@ 9 tcinel  staff   306 18 Aug 23:27 manifests
-rw-------@ 1 tcinel  staff   812 18 Aug 23:27 metadata.json
drwxr-xr-x@ 5 tcinel  staff   170 18 Aug 23:27 spec
drwxr-xr-x@ 3 tcinel  staff   102 18 Aug 23:27 templates
drwxr-xr-x@ 6 tcinel  staff   204 18 Aug 23:27 tests
```

This can cause permissions errors when running `puppet agent` with a Puppet master, e.g:

```
Error: /File[/var/lib/puppet/facts.d]: Failed to generate additional resources using 'eval_generate': Error 400 on SERVER: Permission denied - /etc/puppet/environments/development/modules/iptables/metadata.json
```
Related: [FORGE-163](https://tickets.puppetlabs.com/browse/FORGE-163)

**Comment by alvagante** [comment]
Should be fixed in 2.1.12

**Comment by TimAtLassian** [comment]
Fantastic. Thanks!
